src/modules/ChannelList/dux/reducers.ts

```diff
@@ -135,7 +135,7 @@ export default function channelListReducer(
         const { channel, isMe } = action.payload;
         const { allChannels, currentUserId, currentChannel, channelListQuery, disableAutoSelect } = state;
         let nextChannels = [...allChannels];
-        let nextChannel: GroupChannel = channel;
+        let nextChannel: GroupChannel = currentChannel;
```

Comment:
not sure why this was set to `channel`. if following the init logic for `nextChannels` in the line above, it should be the current state